src/taxonomy/import-tags/index.js

```diff
@@ -0,0 +1,5 @@
+import * as actions from './data/actions';
+
+export {
+  actions, // eslint-disable-line import/prefer-default-export
+};
```

Comment:
It's better to explicitly list what we're exporting here:
```suggestion
// eslint-disable-line import/prefer-default-export
export { importTaxonomy } from './data/actions';
```

And then use it on [TaxonomyListPage](https://github.com/openedx/frontend-app-course-authoring/pull/675/files#diff-60367d61e5f2750d7f4224e061551f5ecf20b56aaddd1ca23ae2852a17f1e424R17) like this:

```javascript
import { importTaxonomy } from './import-tags';
```

Reply:
I was trying to copy the [ADR](https://github.com/openedx/frontend-template-application/blob/master/docs/decisions/0002-feature-based-application-organization.rst) (the reference came from [here](https://jaysoo.ca/2016/02/28/applying-code-organization-rules-to-concrete-redux-code/#module-index-and-constants)), but we will probably only have a second function here. I will change that.

Also, I'll rename `actions` to `utils` to prevent confusion with reducer actions.